code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -210,7 +210,7 @@
 
 /datum/action/innate/cult/blood_spell/horror
 	name = "Hallucinations"
-	desc = "A <u>ranged yet stealthy</u> spell that will break the mind of the victim with nightmarish hallucinations."
+	desc = "Gives hallucinations to a ranged target. A silent and invisible spell."
```

Comment:
You should probably specify that this is a ranged spell in the new description, especially since most of the other cult spells are hand-based/melee-only.